runtime/runtime.go

```diff
@@ -35,3 +35,8 @@ func HTTPMethodFromContext(ctx context.Context) (method string) {
 	method, _ = ctx.Value("http-method").(string)
 	return method
 }
+
+func GetAllowUnknownFromContext(ctx context.Context) (allowUnknown bool) {
+	allowUnknown, _ = ctx.Value("allow-unknown").(bool)
```

Comment:
how do you feel about extracting `http-method` and `allow-unknown` as `const`

Reply:
Yep, sure